src/applications/gi-sandbox/actions/index.js

```diff
@@ -4,7 +4,11 @@ import { api } from '../config';
 
 import { fetchAndUpdateSessionExpiration as fetch } from 'platform/utilities/api';
 
-import { buildSearchFilters, rubyifyKeys } from '../utils/helpers';
+import {
+  buildSearchFilters,
+  rubyifyKeys,
+  searchCriteraFromCoords,
+} from '../utils/helpers';
```

Comment:
Why are these imported?

Reply:
I only imported `searchCriteriaFromCoords` using it in the geolocateUser action